script.js

```diff
@@ -0,0 +1,15 @@
+const hamburger = document.querySelector('.menu-bars');
+const cross = document.querySelector('.cross');
+hamburger.addEventListener('click', () => {
+  const mobileMenu = document.querySelector('.menu');
+  const header = document.querySelector('.header');
+  mobileMenu.style.display = 'flex';
+  header.style.display = 'none';
+});
+
+cross.addEventListener('click', () => {
+  const mobileMenu = document.querySelector('.menu');
+  const header = document.querySelector('.header');
+  mobileMenu.style.display = 'none';
+  header.style.display = 'flex';
+});
```

Comment:
I noticed that this function is not working when I click on any menu option, Let's fix this.
"When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears."

Reply:
- [ ] Kindly, fix this function as it still does not work as the previous reviewer has suggested earlier.